src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    Options: 'Opções',
+    optselectd: 'OPÇÃO SELECIONADA',
+    Oval: 'Oval',
+    overallprogress: 'Progresso geral',
+    Pause: 'Pausar',
+    pclosetag: '<span>Tem certeza que deseja fechar esta enquete?  </span>',
+    pclose: 'Tem certeza que deseja fechar a votação?',
+    Pclosed: 'Enquete fechada',
+    pdfnotrender: 'Há um problema no carregamento da janela, favor recarregar browser.',
+    Pdsuccess: 'Enquete deletada com sucesso',
+    PEdit: 'Editar enquete',
+    PermissionDeniedError: 'Acesso à Webcam foi negado.',
+    PERMISSION_DENIED: 'Você negou acesso à sua Webcam(câmera/microfone).',
+    Play: 'Play',
+    playsessionmsg: 'Clique ‘Play’ para iniciar a sessão.',
+    pleasewaitWhSynNewCont: 'Favor aguardar. Sincronizando novo conteúdo.',
+    plswaitwhile: 'Favor aguardar....',
+    pool: 'Enquete',
+    pollblank: 'Questão não pode ser deixada em branco',
+    pollCancel: 'Fechar',
+    pollDel: 'Tem certeza que deseja deletar esta enquete?',
+    pollHead: 'Votar nesta enquete',
+    pollmaybeshown: 'Enquete será publicada',
+    pollmybpublish: 'Enquete será publicada em breve',
+    pollresult: 'Resultado anterior',
+    ppoll: 'Publicar enquete',
+    PQuestions: 'Questões da enquete',
+    PQuiz: 'Publicar quiz',
+    Precheck: 'Pré-checagem',
+    precheckStart: 'Pré-checagem',
+    prechkcmplt: 'Pré-checagem completa',
+    Presult: 'Resultado da enquete',
+    preWllBshortly: 'Apresentação será compartilhada em breve',
+    Prev: 'Ant',
+    proposedspeed: 'Velocidade proposta',
+    PSetting: 'Configuração da enquete',
+    Publish: 'Publicar',
+    QClosed: 'Quiz fechado',
+    Question: 'Questão',
+    Quiz: 'Quiz',
+    'Quiz/page': 'Quiz / página',
+    Quizes: 'Quizzes',
+    quizmayshow: 'Prepare-se para um quiz',
+    quizreviewpublish: 'Revisar e publicar quiz',
+    RaiseHandStdDisabled: 'Abaixar mão',
+    RaiseHandEnable: 'Mão levantada',
+    raiseHandNotify: 'Estudante(s) com mão levantada',
+    RaiseHandStdEnabled: 'Levantar mão',
+    readonlymode: 'Apenas leitura',
+    receivedVotes: 'Votos recebidos',
+    reclaim: 'Retomar',
+    Rectangle: 'Retângulo',
+    rejected: 'Outra aplicação no seu computador pode estar usando sua Webcam. Favor fechar todas as outras aplicações',
+    recarregar: 'Recarregar',
+    reloadDoc: 'Recarregar documento',
+    removeContext: 'Remover Marcador de Páginas',
+    replay: 'Replay',
+    Replay: 'Replay do começo.',
+    replay_message: 'Obrigado por assistir.',
+    requestScreenShare: 'Pedir compartilhamento de tela',
+    requestedScreenShare: 'Compartilhamento de tela foi pedido',
+    Reset: 'Redefinir',
+    Rtime: 'Tempo restante',
+    rusureCquiz: '<span>Tem certeza que deseja fechar o quiz? </span>',
+    rvtu: 'Votos recebidos/ total de usuários',
+    reply: 'Responder',
+    safariBrowserIssue: 'Seu browser Safari {virtualclass1} não é suportado. Nós suportamos completamente o Chrome e o Firefox',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 128. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)